src/fit-curve.js

```diff
@@ -512,21 +507,17 @@ class maths {
     //https://en.wikipedia.org/wiki/Norm_(mathematics)#Euclidean_norm
     //  var norm = logAndRun(math.norm);
     static vectorLen(v) {
-        var a = v[0], b = v[1];
-        return Math.sqrt(a*a + b*b);
+        return Math.hypot(...v);

```

Comment:
In case you are wondering: `Math.hypot` is basically doing what we need for vector magnitude calculation. It squares all arguments, adds them together and squareroots them. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot

Reply:
alternatively `Math.sqrt(Math.dot(v,v))`

Reply:
I like native solution. nice find.